yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/server/ExternalServer.java

```diff
@@ -201,6 +192,25 @@ private boolean startServer(ProcessBuilder processBuilder) throws IOException, I
         return serverProcess.isAlive();
     }
 
+    private int getAvailablePort(final int unavailablePort) {
```

Comment:
It seems that the start method would only return once the server is fully started therefore the ports are in use (no race condition). Maybe worth while to add a comment to that effect?